spread.yaml

```diff
@@ -276,36 +276,34 @@ suites:
    summary: core24 specific tests
    systems:
      - ubuntu-24.04*
+   environment:
+     SNAPCRAFT_BUILD_ENVIRONMENT: ""
+   prepare: |
+     # Workaround to be able to test the snaps
+     if snap install core24; then
```

Comment:
I take it we won't have a point where it's released to `beta` or `candidate` that we want to check?

Reply:
this as is does no harm, but it is a workaround to have the base as snapd does not install it by default, by the time we reach beta or candidate core24 will look pretty similar across the channel risks (security updates mostly)